hal/src/main/native/include/HAL/DriverStation.h

```diff
@@ -47,6 +47,13 @@ enum HAL_AllianceStationID : int32_t {
   HAL_AllianceStationID_kBlue3,
 };
 
+enum HAL_MatchType {
+  HAL_kMatchType_none,
+  HAL_kMatchType_practice,
+  HAL_kMatchType_qualification,
+  HAL_kMatchType_elimination,
```

Comment:
I think there's one more match type: test.

Reply:
The netcomm header only has these 4, and iirc test matches just run with none for the type.